tests/unit/Step.spec.js

```diff
@@ -0,0 +1,203 @@
+import { mount, createLocalVue, config } from "@vue/test-utils";
+import Step from "@/Components/Step.vue";
+import VueI18n from "vue-i18n";
+import Vuex from "vuex";
+
+describe("Step.vue", () => {
+    let wrapper;
+    beforeEach(() => {
+        const localVue = createLocalVue();
+        localVue.use(VueI18n);
+        localVue.use(Vuex);
+        wrapper = mount(Step, {
+            localVue
+        });
+        const messages = require("@/locales/en.json");
+        const i18n = new VueI18n({
+            locale: "en",
+            fallbackLocale: "en",
+            messages: messages
+        });
+        config.mocks.i18n = i18n;
+        config.mocks.$t = key => {
+            return i18n.messages[key];
+        };
+    });
+
+    it("Has the main div tag", () => {
```

Comment:
It is better to check tags with snapshot testing (`expect(wrapper).toMatchSnapshot`)

Reply:
Thanks for the review. I have added the snapshot tests for the component, please check.

Reply:
> Thanks for the review. I have added the snapshot tests for the component, please check.

Since you have snapshot tests, you don't need the 'Has the main div tag' test.